dev/com.ibm.ws.cdi.weld/src/com/ibm/ws/cdi/impl/managedobject/CDIEJBManagedObjectFactoryImpl.java

```diff
@@ -89,28 +91,86 @@ public ManagedObjectContext createContext() {
         return managedObjectContext;
     }
 
-    @SuppressWarnings("unchecked")
+    @SuppressWarnings("unchecked")	
```

Comment:
This suppression annotation isn't needed, and there's trailing whitespace suggesting the code hasn't been auto-formatted.